### PR TITLE
Add some tests

### PR DESCRIPTION
**Pull request by janisz**
Use `DefaultConfig` in test
Make queue tests logs verbose
Add somne corner case tests

### CODE REVIEW

File: queue/bytes_queue_test.go [hunk 1]
```diff
@@ -226,6 +255,30 @@ func TestGetEntryFromInvalidIndex(t *testing.T) {
...
+
+	// given
+	queue := NewBytesQueue(13, true)
+
+	// when
+	result, err := queue.Get(1)
+
+	// then
+	assert.NotNil(t, result)
+	assert.NoError(t, err)
```
Comment (janisz) [inline queue/bytes_queue_test.go]: Maybe we should return Empty queue error here.

File: queue/bytes_queue_test.go [hunk 2]
```diff
@@ -226,6 +255,30 @@ func TestGetEntryFromInvalidIndex(t *testing.T) {
...
 }
 
+func TestGetEntryFromIndexOutOfRange(t *testing.T) {
+	t.Parallel()
+
+	// given
+	queue := NewBytesQueue(0, true)
+
+	// when
+	assert.Panics(t, func() { queue.Get(42) })
```
Comment (janisz) [inline queue/bytes_queue_test.go]: Maybe we should return error instead of Panicking.

Comment (druminski) [inline queue/bytes_queue_test.go]: sure, we can add some args validation in the constructor. Also args in newBigCache constructor also can be validated, for example MaxEntriesInWindow or MaxEntrySize need to be greater than 0.

File: queue/bytes_queue_test.go (hunk 1 above)
Comment (druminski) [inline queue/bytes_queue_test.go]: good remark, it will be a better behavior in that situation

File: queue/bytes_queue_test.go [hunk 3]
```diff
@@ -71,7 +100,7 @@ func TestAllocateAdditionalSpaceForInsufficientFreeFragmentedSpaceWhereHeadIsBef
 	t.Parallel()
 
 	// given
-	queue := NewBytesQueue(25, false)
+	queue := NewBytesQueue(25, true)
```
Comment (druminski) [inline queue/bytes_queue_test.go]: Output from bench tests is cleaner when verbose mode in unit tests is disabled as bench tests also run unit tests.

File: queue/bytes_queue_test.go (hunk 1 above)
Comment (janisz) [inline queue/bytes_queue_test.go]: Fixed

File: queue/bytes_queue_test.go (hunk 2 above)
Comment (janisz) [inline queue/bytes_queue_test.go]: Fixed

File: queue/bytes_queue_test.go (hunk 3 above)
Comment (janisz) [inline queue/bytes_queue_test.go]: Fixed

**Comment by druminski** [comment]
:+1:
